manifests/init.pp

```diff
@@ -71,6 +71,10 @@
     fail('This module only works on Debian or derivatives like Ubuntu')
   }
 
+  if $::apt_gpgversion and versioncmp ('2.0.26', $::apt_gpgversion) > 0 {
+    warning("You are running an old version of gpg. version=$::apt_gpgversion, please upgrade to at least 2.0.26")
```

Comment:
No. This will throw warnings on Ubuntu Trusty Thar, Debian Wheezy and a few more and users have no way of getting a newer version. Also, Debian and Ubuntu make a point of back porting security fixes from newer releases into older ones if needed, like CVE's so an exact version matching is probably not a correct representation.